demo/README.md

```diff
@@ -35,15 +100,13 @@ Note: This is work in progress.  Some environment variables are hardcoded in the
 - BACKEND_PORT:  Port at which the backend client server is available;  Default value:  3000;  Set in [config.json](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/demo/client/backend/fabric-gateway/config.json)
 
 
-```                 
-cd $FPC_PATH/demo/client/backend/fabric-gateway
-export NETWORK_NAME=fabricfpc-fpc_basic
+```
+export NETWORK_NAME=fabricfpc_basic
 export BACKEND_PORT=3000
-docker run --network $NETWORK_NAME -d  -v ${PWD}:/usr/src/app \
-		 -p $BACKEND_PORT:$BACKEND_PORT --name client_backend auction_client_backend
+docker run --network $NETWORK_NAME -d -p $BACKEND_PORT:$BACKEND_PORT --name auction-client auction_client_backend
```

Comment:
it would seem more natural to start it with docker-compose, in which case the network name and backend-port issue should be moot?

Reply:
I did think about just giving directions for docker-compose directly and not giving directions to bring up each container separately, but I thought since this section is about bringing each individual component this would be better. I also wanted to build on the directions that Sowmya started. 

Saying that I am open to removing these directions and only using docker-compose. The difficulty is I don't think you can use docker-compose to bring about a single service at a time. How do you think we should proceed with these directions? We don't necessarily need to have individualized directions for each component.

Reply:
i'm pretty sure you can manipulate a single service with docker-compose, including starting it up.

Reply:
Oh, that's easy enough. I can update the instructions for that.

Reply:
Updated instructions so that the auction client image is now built and run in one step while using the docker-compose backend. The user will need to provide the COMPOSE_PROJECT_NAME when running the command though.